bindata/manifests/daemon/daemonset.yaml

```diff
@@ -1,4 +1,88 @@
 ---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cni-binary-copy-script
+  namespace: {{.Namespace}}
+  annotations:
+    kubernetes.io/description: |
+      This is a script used to copy CNI binaries based on host OS
+    release.openshift.io/version: "{{.ReleaseVersion}}"
+data:
+  cnibincopy.sh: |-
```

Comment:
Can we add dockerfile/entrypoint.sh in intel/bond-cni repo and use that directly in bond-cni initContainer?